CHANGELOG.md

```diff
@@ -25,6 +25,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 - `opentelemetry-instrumentation-jinja2` Allow instrumentation of newer Jinja2 versions.
 
+### Changed
+- `opentelemetry-instrumentation-pymongo` Add check for suppression key in PyMongo.
+
```

Comment:
Please include the PR # similar to other formatting.

Reply:
done